pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprfocus.nix

```
@@ -8,13 +8,13 @@
 
 mkHyprlandPlugin hyprland {
   pluginName = "hyprfocus";
-  version = "0-unstable-2025-01-04";
+  version = "0-unstable-2025-04-05";
```

```suggestion
  version = "0-unstable-2025-04-01";
```
UTC for https://github.com/daxisunder/hyprfocus/commit/8061b05a04432da5331110e0ffaa8c81e1035725

---

ah for sure, thanks!